cartographer_ros/cartographer_ros/msg_conversion.h

```diff
@@ -17,61 +17,63 @@
 #ifndef CARTOGRAPHER_ROS_GOOGLE_CARTOGRAPHER_SRC_MSG_CONVERSION_H_
 #define CARTOGRAPHER_ROS_GOOGLE_CARTOGRAPHER_SRC_MSG_CONVERSION_H_
 
+#include <boost/array.hpp>
```

Comment:
There are just a few minor uses of boost in cartographer_ros; this is one of them.  I just added the include of the header file to be more correct, since it was having trouble finding the symbols.  While I think removal of boost would be valuable, I think we should do it in upstream (ROS1) cartographer, and then inherit it here.